packages/scripts/src/tasks/providers/appData.ts

```diff
@@ -13,9 +13,12 @@ import {
 } from "../../lib/app-data";
 
 /** Prepare sourcely cached assets for population to app */
-const postProcessAssets = async (options: { sourceAssetsFolders: string[] }) => {
-  const { sourceAssetsFolders } = options;
-  return new AssetsPostProcessor({ sourceAssetsFolders }).run();
+const postProcessAssets = async (options: {
+  sourceAssetsFolders: string[];
+  folderMetadata?: Map<string, { remote?: boolean; folderName?: string }>;
+}) => {
+  const { sourceAssetsFolders, folderMetadata } = options;
+  return new AssetsPostProcessor({ sourceAssetsFolders, folderMetadata }).run();
```

Comment:
We're using `folderMetadata` as an additional arg that only applies to remote assets. It would be better to define a unified structure that can just be used for both core and remote, likely dropping the sourceAssetsFolder string and using the structure from the deployment config instead (which should include the metadata fields anyways) - essentially using 1 data structure instead of 3